packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -37,6 +37,7 @@ This property is similar to the `allowedTypes` property. The difference is the f
 
 - Type: `String`
 - Required: No
+- Platform: Web
```

Comment:
@youknowriad 👋 Could you also be able to review README.md changes here? I believe Platform attribute is being added the first time.

Reply:
Yes, this seems like a good idea to include in the README to raise awareness of the mobile side of things. I don't know if this is going to be clear enough that it's about native mobile apps. We should first provide some guidelines in the official Gutenberg docs about that and ensure that those components can work out of the box when imported as npm packages.

Can we discuss it first at the upcoming weekly Core JS meeting so people could share their opinions? See https://make.wordpress.org/meetings/.

There should also a way to include the same information in JSDoc comments as we are moving towards automated public API generation from such JSDoc comments. @nosolosw  - do you think it would be easy to extend `@wordpress/docgen` to support it?

Reply:
The biggest problem as of now is to have the components' props documented automatically. Most of them don't contain any JSDoc and, AFAIK, we haven't looked into infer anything from the internal structure (such as inferring existing props from React components).

Another thing is that there is no JSDoc mechanism to declare that a particular param is only avaliable for web or mobile. I guess we can add that to the param's description.

Reply:
We discussed this topic during Core JS chat this week:

https://wordpress.slack.com/archives/C5UNMSU4R/p1566914617419700
(link requires registration at https://make.wordpress.org/chat/).

We should proceed with the platform related notes as proposed in this PR. Let's open PR which documents the process so we could replicate it for other components. It usually helps to enforce some standards when they are well documented.

@nosolosw, thanks for chiming in. We definitely will have to find a way to mark with JSDoc whether a given function or components as an entity is available on the given platform. We can start at the general level and leave props for future iterations.

Reply:
Thanks @gziolo ! We'll be starting such a document to describe `Platform` usage in `README.md` files, we can then iterate on it on a separate PR.

Reply:
@gziolo @koke I've created https://github.com/WordPress/gutenberg/issues/17230 to track the discussion about documentation taking into account the core-js conversation. Please, feel free to expand on what did you like about some of the examples provided (ionic, apple docs) or anything else that you find missing.

Reply:
Sweet, thanks @nosolosw  👍